docs/02-computational-graphs/01-scalar.ipynb

```diff
@@ -390,7 +388,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In `mlfz,` the computational graph is fully represented by `Scalar's and `Edge's. All you need to do is to define functions using operations and functions as building blocks. `Scalar` objects are compatible with addition, subtraction, multiplication, division, and exponentiation; that is, the operators `+`, `-`, `*`, `/`, and `**`. \n",
+    "In `mlfz,` the computational graph is fully represented by `Scalars and Edges`. All you need to do is to define functions using operations and functions as building blocks. `Scalar` objects are compatible with addition, subtraction, multiplication, division, and exponentiation; that is, the operators `+`, `-`, `*`, `/`, and `**`. \n",
```

Comment:
Can you change this to `Scalar`s and `Edge`s and move the comma out from `mlfz,`?

Reply:
done